Add post from 23rd session

anezkamll opened the pull request.

anezkamll committed:
Add post from 23rd session

anezkamll committed:
Add missing picture

befeleme commented on content/2019-03-26_dvacaty_treti_sraz.md:
```diff
@@ -0,0 +1,30 @@
+
+### OrderedDict
+
+Doposud jsme v kódu pracovaly se seznamem startovních políček bez jakýchkoliv dalších detailů. Pro hru ale potřebujeme krom polohy startovních políček ještě několik doplňujícíh informací, například kterým směrem jsou orientována, a také potřebujeme, aby záznamy zůstávaly ve stejném pořadí. Karolina se rozhodla předělat seznam na slovník, kde klíčem bude číslo políčka a hodnotami budou směr a souř ...
```
```suggestion
Doposud jsme v kódu pracovaly se seznamem startovních políček bez jakýchkoliv dalších detailů. Pro hru ale potřebujeme krom polohy startovních políček ještě několik doplňujících informací, například kterým směrem jsou orientována, a také potřebujeme, aby záznamy zůstávaly ve stejném pořadí. Karolina se rozhodla předělat seznam na slovník, kde klíčem bude číslo políčka a hodnotami budou směr a souřadnice. Starší verze Pythonu ale náhodně mění pořadí položek ve slovníku, bylo tedy třeba použít knihovnu `Collections` a z ní třídu [OrderedDict](https://docs.python.org/3/library/collections.html#collections.OrderedDict), která starším verzím Pythonu přidává schopnost držet záznamy ve slovníku v takovém pořadí, v jakém jsou do něj vkládány. Od Pythonu 3.6 výše je tato vlastnost slovníků přímo zaintegrovaná do Pythonu. 
```

anezkamll replied:
@befeleme V dokumentaci psali Python 3.7 https://docs.python.org/3/library/collections.html#collections.OrderedDict

befeleme replied:
Ok, podle dokumentace:
> Changed in version 3.7: Dictionary order is guaranteed to be insertion order. This behavior was an implementation detail of CPython from 3.6.

Řídila jsem se tím, že nám to předtím fungovalo i v 3.6. :) Souhlasím i s původní 3.7.

anezkamll committed:
Fix typo

Nešlo přijmout tu suggested change, protože zahrnovala dvě opravy a nejsem si jistá, jak to je teda s tím OrderedDict, tak jsem překlep opravila zvlášť.
